src/command/backup/backup.c

```diff
@@ -1474,6 +1474,18 @@ backupJobResult(
                 {
                     LOG_DETAIL_PID_FMT(
                         processId, "match file from prior backup %s (%s)%s", strZ(fileLog), strZ(logProgress), strZ(logChecksum));
+
+                    // File size may need to be updated because the file original file size may have been larger than what was found
```

Comment:
`file original file size` sounds weird to me :thinking: `because the original file size` should be enough?

Reply:
Definitely a typo. Fixed in 69780b86063ff3.

Reply:
cool, Thanks. LGTM now :+1: